src/Tests/Unit/Config/MaxLoginConfig.test.ts

```
@@ -208,83 +219,71 @@ describe('MAX_CONFIG', () => {
-
   describe('possibleResults.invalidPassword', () => {
-    it('returns true when popupWrongDetails element present', async () => {
-      MOCK_ELEMENT_PRESENT_ON_PAGE.mockResolvedValue(true);
-      const invalidPwCheckers = MAX_CONFIG.possibleResults.invalidPassword ?? [];
-      const checker = invalidPwCheckers[0] as (opts: { page: Page }) => Promise<boolean>;
+    it('returns true when error text is visible', async () => {
       const page = makeMockPage();
+      const errorLoc = makeMockLocator(true);
+      MOCK_GET_BY_TEXT.mockReturnValue(errorLoc);
+      const invalidPwCheckers = MAX_CONFIG.possibleResults.invalidPassword ?? [];
+      const checker = invalidPwCheckers[0] as ({ page }: { page: Page }) => Promise<boolean>;
```

ESLint

[error] 'page' is defined but never used.

(no-unused-vars)